docs/api.rst

```diff
@@ -9340,6 +9340,118 @@ Editing blockchain account data
    :statuscode 409: An account given to edit does not exist or a given tag does not exist.
    :statuscode 500: Internal rotki error
 
+
+Chain agnostic account operations
+==================================
+
+.. http:patch:: /api/(version)/blockchains/agnostic/accounts
+
+   .. note::
+      Supported blockchains types: ``EVM, BITCOIN, SUBSTRATE, EVM_LIKE``
```

Comment:
From the api's point of view we need EVM and EVM like to be one choice. Touching lefteris.eth in all evm and evmlike chains.